contracts/src/fp/OPSuccinctFaultDisputeGame.sol

```diff
@@ -124,6 +127,9 @@ contract OPSuccinctFaultDisputeGame is Clone, ISemver {
     ///         is the amount of the bond that is distributed to the prover when proven with a valid proof.
     uint256 internal immutable PROOF_REWARD;
```

Comment:
Shouldn't there be different rewards? Shouldn't these be configurable in the entrypoint?